test/consent-string.test.js

```diff
@@ -29,6 +29,20 @@ describe('ConsentString', function () {
     expect(new ConsentString(consentString.getConsentString(false))).to.deep.include(consentData);
   });
 
+  it('has a getMetadataString method and decodeMetadaString static method', function () {
```

Comment:
Can you also add a test that validates a fixed/pre-defined metadata string? (both for encoding and decoding)
That helps making sure the library is compatible with spec and not just with itself.